.github/workflows/python-tests.yml

```
@@ -54,7 +54,9 @@ jobs:
 
       - uses: actions/checkout@v4
       - name: Free Disk Space (Ubuntu)
+        continue-on-error: true
         uses: jlumbroso/free-disk-space@main
+        continue-on-error: true          # Self-hosted runner: bash arithmetic fails on large disks (scientific notation)
```

actionlint (1.7.11)

[error] key "continue-on-error" is duplicated in element of "steps" section. previously defined at line:57,col:9

(syntax-check)